src/man/sssd.conf.5.xml

```diff
@@ -2030,6 +2030,18 @@ p11_uri = library-description=OpenSC%20smartcard%20framework;slot-id=2
             configuration section, that is, in a section called
             <quote>[domain/<replaceable>NAME</replaceable>]</quote>
             <variablelist>
+                <varlistentry>
+                    <term>enabled</term>
+                    <listitem>
+                        <para>
+                            Override the enabled/disabled state of the <quote>domains</quote>
+                            list at <quote>[section]</quote>. Set it to <quote>true</quote>
```

Comment:
[sssd] instead of [section]? Also please keep withing 80 chars limit.

Reply:
Thanks! Done